MEAnalysis/crab_vhbb/heppy_crab_script.py

```diff
@@ -44,8 +44,11 @@ def getLumisProcessed(vlumiblock):
     if arg.startswith("ME_CONF="):
         me_conf_name = arg.split("=")[1]
 
-JobNumber=sys.argv[1]
-print "JobNumber", JobNumber
+try:
```

Comment:
ok, I think we had a cmdline flag for local testing which disabled some crab-specific stuff.